core/src/main/java/com/linecorp/armeria/common/multipart/DefaultMultipart.java

```diff
@@ -173,7 +173,7 @@ public boolean isOpen() {
 
     @Override
     public boolean isEmpty() {
-        return parts.isEmpty();
+        return false;
```

Comment:
I left why I changed this on the commit message:
> Change to return false from StreamMessage.isEmpty() when the StreamMessage is created from a empty Multipart
Because it produces the ending boundary.

Reply:
Oops, it should only be `false` when `StreamMessage.isEmpty()` is called. Let me fix that. 😉

Reply:
This method is only called after a `Multipart` is converted into a `StreamMessage` and it's not always empty because it produces at least a closing boundary. So let me add a comment and keep that as is.

Reply:
Understood. 👍